app/config/collections.php

```diff
@@ -2248,21 +2248,21 @@
                 '$id' => ID::custom('_key_runtime'),
                 'type' => Database::INDEX_KEY,
                 'attributes' => ['runtime'],
-                'lengths' => [2048],
+                'lengths' => [768],
```

Comment:
do we want to drop these back down, and has this been tested against Mongo?

Reply:
Yes this is to ensure support for MySQL. MySQL has some limits on the maximum index length. 
I don't think it has been tested on Mongo, @fogelito can confirm. 
Does mongo use index lengths ?

Even if that were the case, it wouldn't be an issue for Appwrite 1.1 👍

Reply:
I Do not know I have a test for this on the new Database Tag and Mongo was passing , but better check